index.html

```diff
@@ -7,38 +7,41 @@
     <!-- fugaz & lato font -->
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap&family=Fugaz+One&display=swap"rel="stylesheet"/>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&display=swap&family=Fugaz+One&display=swap"
+      rel="stylesheet"
+    />
     <!-- icon link -->
     <link rel="stylesheet" href="css/main.css" />
     <script
       src="https://kit.fontawesome.com/59c4622193.js"
       crossorigin="anonymous"
     ></script>
-    <title>ELGALLOGIRO</title>
+    <title>EL GALLO GIRO</title>
```

Comment:
A lot of stuff in here is going to be formatting. Once I fixed the broken divs in your slider section this page was able to format itself. I'll leave comments next to any meaningful changes.